app/functions.R

```diff
@@ -105,8 +106,8 @@ renderScoreHtml <- function(risk) {
     formatDynamicString(round(score)), 
     case_when(
       score<30 ~ paste0(
-        ", which is (relatively) safe. Even so, it's a good time to make sure that you know how to ",
-        tags$a("protect yourself & others. ", href = urls$cdc_get_ready)),
+        ", which is (relatively) safe. Even so, it's a good time to make sure that you're ",
```

Comment:
@pendlrobinson I wonder if we should be hitting the different api url based on the country selection. Right now in line 34 we are hitting urls$covid_score_api, which points to the US endpoint.

Reply:
@xindyhuI currently we can use the US endpoint to access the Belgium data. I am not sure if we should be using different endpoints based on the country selection. @skirk-mpr or @shruthirames do you have any thoughts about this?

Reply:
Hey @pendlrobinson  @xindyhu !

So right now the exact same application code/logic is deployed to all the endpoints/Lambdas regardless of the region the API is physically located in (US vs. EU).  I believe the intention of having endpoints deployed in EU was not to have a different calculation deployed to it but for GDPR compliance and some potential performance improvement for the client also located in the EU.  (_Theoretically_, they would be physically closer to the endpoint they make the request to, so there could be lower latency in the request/response times.)  The only other notable difference is that the API Keys are region specific, so VITOs API Key would only work with requests made to the EU API Gateway that the PROD-EU endpoint/URL is deployed behind. 

Is there a need or requirement (now or foreseeable in the future) to configure the EU endpoint to only handle request for a specific set of country codes (or the US endpoints to only be responsive to a US county code request)?  That's definitely something that could be achieved through some configuration in the application and deployment process - but it would make the endpoints less flexible and might introduce some maintenance issues since the endpoints would need to be deployed and configured differently.  Happy to discuss though!

@shruthirames  any thoughts from your end regarding this topic?